lndc/README.md

```diff
@@ -0,0 +1,19 @@
+lndc
+==========
+
+The lndc package implements a secure crypto messaging protocol based off of
+the [Noise Protocol Framework](http://noiseprotocol.org/noise.html). The
+package exposes the raw state machine that handles the handshake and subsequent
+message encryption/decryption scheme. Additionally, the package exposes a
+[net.Conn](https://golang.org/pkg/net/#Conn) and a
+[net.Listener](https://golang.org/pkg/net/#Listener) interface implementation
+which allows the encrypted transport to be seamlessly integrated into a
+codebase.
+
+The secure messaging scheme implemented within this package uses `NOISE_XX` as the handshake for authenticated key exchange.
```

Comment:
Should mention here that this is not what the brontide code does and it isn't compatible

Reply:
Thanks, will do.